internal/remote/user.go

```diff
@@ -5,13 +5,13 @@ import (
 	"encoding/gob"
 	"errors"
 	"fmt"
+	"github.com/ProtonMail/gluon/internal/queue"
```

Comment:
Could you run goimports on this file? Import ordering is a bit off.

Reply:
Done.